### PR TITLE
Setup redux

### PR DESCRIPTION
**Pull request by AnnaVih**

### CODE REVIEW

File: app/actions/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export {
```
Comment (webtechalex) [inline app/actions/index.js]: can we move redux-related files into a 'redux' folder? so: `app/redux/reducers...` `app/redux/actions...`

File: app/store.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { createStore, applyMiddleware, compose } from 'redux';
```
Comment (webtechalex) [inline app/store.js]: Thanks for encapsulating store creation away from the rest of redux/react. I'd like us to make some of the decisions about this all together. 👍 Thunk or Saga, for example?

**Review by webtechalex — CHANGES_REQUESTED** [review]

File: app/store.js (hunk 2 above)
Comment (AnnaVih) [reply app/store.js]: Agree! =)

Comment (webtechalex) [reply app/store.js]: Suggest moving the store file into the redux folder.

**Review by webtechalex — CHANGES_REQUESTED** [review]

File: app/store.js (hunk 2 above)
Comment (AnnaVih) [reply app/store.js]: @webtechalex Done! =)

**Review by webtechalex — APPROVED** [review]

**Pull request merged**